gradio/queueing.py

```diff
@@ -461,74 +459,6 @@ def get_status(self) -> EstimationMessage:
             queue_size=len(self),
         )
 
-    async def call_prediction(self, events: list[Event], batch: bool) -> dict:
-        body = events[0].data
-        if body is None:
-            raise ValueError("No event data")
-        username = events[0].username
-        body.event_id = events[0]._id if not batch else None
-        try:
-            body.request = events[0].request
-        except ValueError:
-            pass
-
-        if batch:
-            body.data = list(zip(*[event.data.data for event in events if event.data]))
-            body.request = events[0].request
-            body.batched = True
-
-        app = self.server_app
-        if app is None:
-            raise Exception("Server app has not been set.")
-        api_name = "predict"
-
-        fn_index_inferred = route_utils.infer_fn_index(
-            app=app, api_name=api_name, body=body
-        )
-
-        gr_request = route_utils.compile_gr_request(
-            app=app,
-            body=body,
-            fn_index_inferred=fn_index_inferred,
-            username=username,
-            request=None,
-        )
-        assert body.request is not None  # noqa: S101
-        root_path = route_utils.get_root_url(
-            request=body.request, route_path="/queue/join", root_path=app.root_path
-        )
-        try:
-            output = await route_utils.call_process_api(
-                app=app,
-                body=body,
-                gr_request=gr_request,
-                fn_index_inferred=fn_index_inferred,
-                root_path=root_path,
-            )
-        except Exception as error:
-            show_error = app.get_blocks().show_error or isinstance(error, Error)
-            traceback.print_exc()
-            raise Exception(str(error) if show_error else None) from error
```

Comment:
This makes it so that `show_error` has no effect now. Tried with this demo

```python
import gradio as gr
#from foo import BAR
#
def calculator(num1, operation, num2):
    if operation == "add":
        return num1 + num2
    elif operation == "subtract":
        return num1 - num2
    elif operation == "multiply":
        return num1 * num2
    elif operation == "divide":
        if num2 == 0:
            raise ValueError("Cannot divide by zero!")
        return num1 / num2

demo = gr.Interface(
    calculator,
    [
        "number", 
        gr.Radio(["add", "subtract", "multiply", "divide"]),
        "number"
    ],
    "number",
    examples=[
        [45, "add", 3],
        [3.14, "divide", 2],
        [144, "multiply", 2.5],
        [0, "subtract", 1.2],
    ],
    title="Toy Calculator",
    description="Here's a sample toy calculator. Allows you to calculate things like $2+2=4$",
)

if __name__ == "__main__":
    demo.launch(show_error=False)

```

Reply:
good catch! fixed now